main/src/http/http_service.rs

```diff
@@ -141,6 +165,26 @@ impl HttpService {
             .or(self.prom_remote_write())
     }
 
+    fn routes_query(
+        &self,
+    ) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+        self.ping()
+            .or(self.query())
+            .or(self.metrics())
+            .or(self.print_meta())
+            .or(self.prom_remote_read())
+    }
+
+    fn routes_store(
+        &self,
+    ) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+        self.ping()
+            .or(self.write_line_protocol())
```

Comment:
Write  operation have to distributed routing; driectly write will result to read abnormal.

Reply:
write request routing to tskv write coodinator